frameworks/sveltekit.md

```diff
@@ -31,7 +31,7 @@ For any static assets (such as images) you will need to add a folder to your pro
 
 ```javascript header=false
 import { vitePreprocess } from "@js/kit/vite";
```

Comment:
This looks like an error too, and in the `@type` line below. I'll clean this up.